src/Entity/ShipmentServiceOptions.php

```diff
@@ -91,9 +97,9 @@ public function __construct($response = null)
             }
             if (isset($response->DirectDeliveryOnlyIndicator)) {
                 $this->DirectDeliveryOnlyIndicator = $response->DirectDeliveryOnlyIndicator;
```

Comment:
You're right that types should be strict. This would allow that:
```php
$this->DirectDeliveryOnlyIndicator = $response->DirectDeliveryOnlyIndicator ? true : false;
```